client/packages/cli/index.ts

```diff
@@ -191,7 +243,7 @@ class CircuitCLI {
         // Check we have an config for each SideEffect
         data.sideEffects.forEach(effect => {
             if (!config.gateways.find(entry => entry.id === effect.target)) {
-                console.error(`Error: Gateway for SideEffect ${effect.type} not found!`)
+                logger.info(`Error: Gateway for SideEffect ${effect.type} not found!`)
```

Comment:
could have extra kwarg {sideeffect: effect.type)
this way we could see and visualise easily which are not found

Reply:
Ahaig, this is user facing logs, so no field formatting is needed